src/dynamics/initial_conditions.jl

```diff
@@ -302,6 +302,10 @@ Parameters are $(TYPEDFIELDS)"""
     m::Int = 4
     ω::Float64 = 7.848e-6
     K::Float64 = 7.848e-6
+    Ω::Float64 = 7.292e-5
+    a::Float64 = 6.37122e6
+    g::Float64 = 9.80616
+    h0::Float64 = 8e3
```

Comment:
there is technically `model.atmosphere.layer_thickness` which is used for the shallow water model, but as you correctly found out, the $\eta$ / `pres` in the model is actually the anomaly (like sea surface height / interface displacement), so you shouldn't add it anyway.